settings.gradle.kts

```diff
@@ -2,19 +2,24 @@
 
 rootProject.name = "avito-android-infra"
 
+val useCompositeBuild: String by settings
+
 include(":samples:test-app")
 include(":samples:test-app-kaspresso")
-include(":samples:test-app-second")
+if (useCompositeBuild.toBoolean()) {
+    include(":samples:test-app-without-backward-compatibility")
```

Comment:
It's an important change for working flow. Without that, we have no ability to create module and add it to sample app because it will break configuration on the previous binary version where new module is absent.
I renamed module and consider to use it with breaking changes and new modules if need

Reply:
But what are you trying to achieve? 
Use new breaking API right away, without new infra version?
Is snackbar change even breaking?

Reply:
I'm trying to use new snackbar modules. Without that change, I will fail on configuration time with the previous binary version

Reply:
ok, i understand, but this will be outdated right after release